packages/mask/src/plugins/FileService/SNSAdaptor/components/Upload.tsx

```diff
@@ -112,6 +114,39 @@ export const Upload: React.FC = () => {
                     label={t('plugin_file_service_use_cdn')}
                 />
             </section>
+
+            <section className={classes.checkItems}>
+                <FormControlLabel
+                    control={
+                        <Radio
+                            color="secondary"
+                            checked={useProvider == 'arweave'}
+                            onChange={(event) => setUseProvider('arweave')}
```

Comment:
```suggestion
                            onChange={() => setUseProvider('arweave')}
```

plz remove unused variable statement.